### DISCUSSION TITLE
Problems about specific clearance in Kidney

### DISCUSSION THREAD

**Discussion by xiangetang** [Open-Systems-Pharmacology/Forum#1721] [Questions & Problems]
My drug was excreted by kidney minorly, and the amount of parent drug collected from urine accounted for 3.3%  of administered dose, i.e. fe=3.3%, and I already known the toltal  plasma clearance calculated by NCA method, which was 9.7 L/h (bioavailability was colse to 1 as well as the absorption fraction in PBPK model ). Therefore, I get the renal plasema clearace=9.7*3.3%, as I put it in the following picture. However, the drug excreated by urine was about 30% in Pksim simulaiton profile. I don't know why? how was the specfic clearance of kidney used to calculted kidney ecreation fraction?
![36d2284a3cd2f8c30a6b8d0f566cdf9](https://github.com/Open-Systems-Pharmacology/Forum/assets/98267394/4a136e9e-acb8-4d24-9ea2-31dac99b8cc1)
![857ba5827d1c55e3444b51f17cfeea9](https://github.com/Open-Systems-Pharmacology/Forum/assets/98267394/989ca5bb-31b9-4beb-be71-d7de52ff6fed)

**Comment by xiangetang** [comment]
Any help will be greatly appreaciated!

**Comment by SCZ1174** [comment]
It is recommended to use the "Parameter Identification" module directly to determine the "specific clearance" of the kidney by fitting the fe.

**Reply by prvmalik** [reply]
Use fe as an observed data point
